Source/UserManagement/deploy/dev-deployment.yaml

```diff
@@ -44,6 +44,12 @@ spec:
         image: karolikl/cbs-usermgmt-backend:$BUILD_ID
         ports:
         - containerPort: 80
+        env:
```

Comment:
The [boundedContext]/deploy/*.yaml files should be removed as all deploy files can now be found in the /Deploy/templates folder. Sorry for not having cleaned this up! 
Can you add these environment variables to https://github.com/IFRCGo/cbs/blob/master/Deploy/templates/usermanagement.yaml instead?